styleguide/source/_patterns/00-atoms/04-images/09-landscape-175x105.twig

```diff
@@ -0,0 +1 @@
+<img {% if class %}class='{{ class }}'{% endif %} src="{{ src|default('https://ipsumimage.appspot.com/175x105') }}" alt="This is an image.">
```

Comment:
Suggestion to use existing image atom and pass the needed unique URL via 'src', and remove this new atom (sorry!)